storage/storage-interface/src/cached_state_view.rs

```diff
@@ -89,6 +91,12 @@ pub struct CachedStateView {
     proof_fetcher: Arc<AsyncProofFetcher>,
 }
 
+impl Debug for CachedStateView {
+    fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
+        write!(f, "{:?}::{:?}", self.id, &self.state_cache)
```

Comment:
I hope DashMap::debug doesn't print out all the KVs..

Reply:
You are right, removed it.